src/styles/blocks/main.scss

```diff
@@ -0,0 +1,102 @@
+.page {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --star-size: 16px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 408px;
```

Comment:
No need to write height for card, it will be better if the height depends on the content inside the card
```suggestion
```